### PR TITLE
first draft

### PR DESCRIPTION
**Pull request by markedwardmurray**
new code-along to follow https://github.com/learn-co-curriculum/reading-ios-nslog

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,5 +2,89 @@
...
+1. Use `NSLog()` to print a string literal to the console.
+2. Practice string formatting by using `NSLog()` to print interpolated strings to the console.
+
+![](https://curriculum-content.s3.amazonaws.com/ios/ios-objc-fundamentals-unit/dalek_interpolate.jpg)
+
 ## Instructions
 
+Fork and clone this lab. Open the `objc-interpolate.xcodeproj` file and navigate to the `FISAppDelegate.m` file. 
+
+Write your code inside the `application:didFinishLaunchingWithOptions:` method before the `return YES;` statement.
```
Comment (jmburges) [inline README.md]: maybe a screenshot of where this is? It's hard for people to find it!

File: README.md [hunk 2]
```diff
@@ -2,5 +2,89 @@
...
+
+### A. Dalek
+
+1. Use an `NSLog()` to print a string that reads "Dalek".
+  * `NSLog(@"Dalek");`
+  * Run the program with `⌘` `R`, this should print `Dalek`.
```
Comment (jmburges) [inline README.md:25]: Screenshot on where the output is?

File: README.md (hunk 1 above)
Comment (jmburges) [inline README.md]: also make a note BEFORE THE `return YES`. so many ppl make that mistake

**Comment by jmburges** [comment]
just needs a few extra screenshots and :shipit:

File: README.md (hunk 1 above)
Comment (markedwardmurray) [inline README.md]: I bolded the existing in-line note and added a screenshot.

**Pull request merged**